src/vm.c

```diff
@@ -88,15 +88,18 @@ internal WrenForeignMethodFn VM_bind_foreign_method(
 internal char* VM_load_module(WrenVM* vm, const char* name) {
   ENGINE* engine = (ENGINE*)wrenGetUserData(vm);
   MAP moduleMap = engine->moduleMap;
-  if (strncmp("./", name, 2) != 0) {
-    if (DEBUG_MODE) {
-      ENGINE_printLog(engine, "Loading module %s\n", name);
```

Comment:
Why was this logging removed?

Reply:
In the og file, there are two possible returns: first, look up if the module is not a custom one by matching `./` and debug log the (builtin) module, and second, look up the filesystem otherwise and log the module. 

What I did in my commit is get rid of one of those logs and make logging happen before any of this, so that we have one log, no matter where we get the module from.